clients/go/grpc/grpc.go

```diff
@@ -0,0 +1,75 @@
+package grpc
+
+import (
+	"context"
+
+	pb "go.buf.build/odpf/gw/odpf/proton/odpf/raccoon/v1beta1"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/google/uuid"
+	raccoon "github.com/odpf/raccoon/clients/go"
+	"github.com/odpf/raccoon/clients/go/serializer"
+)
+
+// NewGrpc creates the new grpc client with provided options.
+func NewGrpc(options ...GrpcOption) (*GrpcClient, error) {
+	gc := &GrpcClient{
+		Serialize:   serializer.PROTO,
```

Comment:
@AkbaraliShaikh does it have to be the public method?

Reply:
no, I will adjust the protection level accordingly for all the options.

Reply:
updated